downstream.go

```diff
@@ -913,6 +879,49 @@ func (dc *downstreamConn) handleMessageRegistered(msg *irc.Message) error {
 				})
 			}
```

Comment:
A `break` is missing here, to prevent the `!uc.isChannel(upstreamName)` check to send an error.